lib/cfn-nag/custom_rules/EMRSecurityConfigurationEncryptionsEnabledAndConfiguredRule.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class EMRSecurityConfigurationEncryptionsEnabledAndConfiguredRule < BaseRule
+  def rule_text
+    'EMR SecurityConfiguration should enable and properly configure encryption at rest and in transit.'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W61'
+  end
+
+  def audit_impl(cfn_model)
+    violating_emr_sec_configs = cfn_model.resources_by_type('AWS::EMR::SecurityConfiguration').select do |sec_config|
+      bad_security_config?(sec_config)
+    end
+
+    violating_emr_sec_configs.map(&:logical_resource_id)
+  end
+
+  private
+
+  def bad_security_config?(security_config_object)
+    # Poorly formatted SecurityConfiguration
+    return true unless security_config_object.securityConfiguration['EncryptionConfiguration']
+
+    encryption_config = security_config_object.securityConfiguration['EncryptionConfiguration']
+
+    # Either encryption type disabled
+    return true unless encryption_config['EnableAtRestEncryption'] && encryption_config['EnableInTransitEncryption']
+
+    bad_at_rest_encryption?(encryption_config)
```

Comment:
do we need a disjunction here... the return value for this will be lost in the ether

Reply:
Ah, good call.  My brain thought that was a return, but the boolean would get lost.  I reworked this, the JSON parsing logic, and the rspec tests to return correct values for all incorrect scenarios.